### PR TITLE
about site worket but medium attractive

### PR DESCRIPTION
**Pull request by neoed**

### CODE REVIEW

File: training-service/src/app/pages/about/about.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  </div>
+<div class="container">
+    <div class="row">
+      <div class="col-sm">
+        W INQOO doskonale wiemy, że jest czas na naukę, ale i na odpoczynek. Jeśli będziesz miał ochotę możemy zagrać razem w FIFĘ, piłkarzyki, urządzić zawody, czy po prostu pochillować w wygodnych fotelach, słuchając muzyki.
+
+Masz inne pomysły? Śmiało! Jesteśmy otwarci i nie ma dla nas życzeń niemożliwych do spełnienia 
+      </div>
+      <div class="col-sm">
+        <img src="./assets/images/aboutUs.jpg" alt="inqoo ekipa" style="display: table; margin: auto"/>
```
Comment (holistic-fox) [inline training-service/src/app/pages/about/about.component.html]: Please do not use inline styles

File: training-service/src/app/pages/about/about.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.css']
+})
+export class AboutComponent implements OnInit {
+
+  constructor() { }
```
Comment (holistic-fox) [inline training-service/src/app/pages/about/about.component.ts]: Please remove constructor and ngOninit

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/pages/about/about.component.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.aboutImg{
```
Comment (holistic-fox) [inline training-service/src/app/pages/about/about.component.css]: this class is empty, please remove

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/pages/about/about.component.ts (hunk 2 above)
Comment (neoed) [reply training-service/src/app/pages/about/about.component.ts]: done

File: training-service/src/app/pages/about/about.component.html (hunk 1 above)
Comment (neoed) [reply training-service/src/app/pages/about/about.component.html]: done

**Review by holistic-fox — APPROVED** [review]

**Pull request merged**